lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -519,19 +519,45 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
   }
 
   // Get all signups.
-  $signup_nids = dosomething_signup_get_signup_nids_by_uid($uid);
+  if (variable_get('rogue_collection', FALSE)) {
+    $northstar_id = dosomething_user_get_northstar_id($uid);
+
+    $rogue_signups = dosomething_rogue_get_activity([
+      'filter' => [
+        'northstar_id' => $northstar_id,
+      ],
+      'limit' => 300,
```

Comment:
Chose to do `limit` query param here instead of using paginated links to only make one API call each time. There are less than 300 campaigns so chose this number for now as a user can only sign up for a campaign once.